### ISSUE TITLE
Django: Completing the move to flake8-docstrings

### ISSUE THREAD

**Issue by jerbob** [python-discord/site#192]
We've decided to add `flake8-docstrings` to project requirements for this repository.
We've agreed on the following list of errors to ignore:
```sh
ignore=
    P102,B311,W503,E226,S311,
    # Missing Docstrings
    D100,D104,D107,
    # Docstring Whitespace
    D202,D203,D204,D212,D214,D215,
    # Docstring Quotes
    D301,D302,
    # Docstring Content
    D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413, D414
```

**Comment by lemonsaurus** [comment]
This dependency should just be added to the Django site, and then we should solve the linting problems there instead. As such, this ticket is open to anyone who wants it, and should be a pretty easy one to get done.

**Comment by gdude2002** [comment]
I'll take this one - should this also apply to the tests?

**Comment by jchristgit** [comment]
I don't see much value in documenting things like nested classes (such as `Meta` from Django or DRF) as these are more or less done entirely in DRF

**Comment by lemonsaurus** [comment]
there is an ignore for nested classes (`D106`), but that would apply to all nested classes everywhere.

Personally I'm not super bothered by that, as nesting classes is quite unusual anyway.

Out of curiosity @gdude2002, how much of a difference would adding D106 make to the linting job?

**Comment by gdude2002** [comment]
With D106 included:

* with tests and manage.py: 273
* without tests or manage.py: 92

With D106 ignored:

* with tests and manage.py: 256
* without tests or manage.py: 75

**Comment by lemonsaurus** [comment]
not that big a difference, then.

**Comment by gdude2002** [comment]
I've left a comment on #215, because I don't 100% understand the code that's being documented here.

**Comment by sco1** [comment]
This has been merged by #215, and updated rules will be merged by #229
